package.json

```diff
@@ -85,6 +88,7 @@
     "@web/test-runner-puppeteer": "^0.10.5",
     "@webcomponents/webcomponentsjs": "^2.5.0",
     "del": "^6.1.0",
```

Comment:
We don't need this in dependency list anymore (del-cli already has this). Please also import in build.js